### PR TITLE
add AutoInstallStrategy for sentry-graphql-22

### PR DESCRIPTION
**Pull request by lbloder**
## :scroll: Description
Add Auto-Install strategy for graphql-22

## :bulb: Motivation and Context
Fixes https://github.com/getsentry/sentry-java/issues/3756

## :pencil: Checklist

- [ ] I reviewed the submitted code
- [ ] I added tests to verify the changes
- [ ] I updated the docs if needed
- [ ] No breaking changes

### CODE REVIEW

File: plugin-build/src/main/kotlin/io/sentry/android/gradle/autoinstall/graphql/Graphql22InstallStrategy.kt
Comment (adinauer) [inline plugin-build/src/main/kotlin/io/sentry/android/gradle/autoinstall/graphql/Graphql22InstallStrategy.kt:30]: We can remove the comment above

**Review by adinauer — APPROVED** [review]
LGTM, we should wait for v8 final release before merging, or do a beta release of the plugin

**Review by romtsn — APPROVED** [review]

**Pull request merged**
